src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -72,4 +76,6 @@ FlexMicroform.defaultProps = {
   disabled: false,
 };
 
-export default FlexMicroform;
+export default connect(
+  updateCaptureKeySelector,
+)(FlexMicroform);
```

Comment:
nit: It seems like this `connect` is unnecessary given that the prop is passed in explicitly here: https://github.com/edx/frontend-app-payment/pull/407/files#diff-a47751e466a3c91d8c2ab6f1f56818edL214